### ISSUE TITLE
QField (Desktop/Cable Offline Mapping)- Not able to extract the updated shapefile from the mobile phone (using QField App)

### ISSUE THREAD

**Issue by Btejeswari** [opengisch/QField#3356]
### Describe the issue
[The aim of the project was to mark the plot boundaries within the study area. A QGis (Version 3.26.1) project (consisting of a raster file and an editable polygon shapefile) was initially set up in the Windows10 OS system. The packaging of the project was done using QFieldSync and this packaged folder has been copied in the mobile phone (Android) device. Here in mobile, using QField App, the project was opened and both the raster & the vector files were accessed. We were able to edit the polygon file at the survey site. After our visit, we have copied this updated project folder from our devices to the System and imported into our QGis project. The entire updated shapefile data went missing. But we were able to find it in our mobile devices.

Initially while testing the entire exercise was successful using Ubuntu OS system. but this issue was faced when we used Windows OS System for our actual project. Currently, the entire field updations are only in the mobile phones. Not able to extract/copy this shapefile from the mobile devices to our systems.]

**Comment by nirvn** [comment]
@Btejeswari , a bit more information is required to figure this one out :) what Android version is your device running on? Have you imported the project into QField once you copied the project files via USB cable?

For Android >= 9, QField has to import your project files into its own app files directory, any edits done to the shapefile would be applied to the imported copy of the shapefile. You would find the up-to-date project and its dataset(s) at this location: <device root>/Android/data/ch.opengis.qfield/files/Imported Projects/.

**Comment by nirvn** [comment]
@Btejeswari , a bit more information is required to figure this one out :) what Android version is your device running on? Have you imported the project into QField once you copied the project files via USB cable?

For Android >= 9, QField has to import your project files into its own app files directory, any edits done to the shapefile would be applied to the imported copy of the shapefile. You would find the up-to-date project and its dataset(s) at this location: <device root>/Android/data/ch.opengis.qfield/files/Imported Projects/.

**Comment by Btejeswari** [comment]
Dear Sir

1Q)- what Android version is your device running on?------ I have been
using Mi POCO X2 Android Model for the project.
2Q)- Have you imported the project into QField once you copied the project
files via USB cable? ------ Yes Sir... the error was noticed once the file
has been imported.

I have searched in this path ---------
/Android/data/ch.opengis.qfield/files/Imported Projects/.  I found my files
in here. Happy to see that the entire updated data (edited Shapefile on the
field) is found. Thank you so much sir for resolving this problem...

Thank you for the reply...

Tejeswari
